MAUI/Segmented-Control/display-mode.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: Display options in .NET MAUI Segmented control | Syncfusion
+description: Learn about the Display options in Syncfusion .NET MAUI Segmented(SfSegmented) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfSegmentedControl
+documentation: ug
+---
+ 
+# Display options in .NET MAUI Segmented (SfSegmented) control
+
+Depending on the application, different scenarios may require icons, text, or a combination of both for effective communication. The segmented control supports these three options.
+
+## Text
+By default, the segmented items is diplayed as text
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:local="clr-namespace:SfSegmentSample"
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+
+    <ContentPage.BindingContext>
+        <local:ViewModel/>
+    </ContentPage.BindingContext>
+
+        <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                    ItemsSource="{Binding ItemCollection}">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    ObservableCollection<SfSegmentItem> itemList = new ObservableCollection<SfSegmentItem>();
+
+    public MainPage()
+    {
+        InitializeComponent();
+        ViewModel viewModel = new ViewModel();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.ItemsSource = viewModel.ItemCollection;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% highlight C# tabtitle="ViewModel.cs" %}
+
+public class ViewModel
+{
+    private ObservableCollection<SfSegmentItem> itemCollection;
+
+    public ViewModel()
+    {
+        itemCollection = new ObservableCollection<SfSegmentItem>()
+        {
+                new SfSegmentItem() {Text="Day"},
+                new SfSegmentItem() {Text="Week"},
+                new SfSegmentItem() {Text="Month"},
+                new SfSegmentItem() {Text="Year"},
+        };
+    }
+
+    public ObservableCollection<SfSegmentItem> ItemCollection
+    {
+        get { return itemCollection; }
+        set { itemCollection = value; }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Image
```

Comment:
Image
Create segmented control with segments having the given  images

Reply:
content changed